.JuliaFormatter.toml

```diff
@@ -6,3 +6,5 @@ import_to_using = false
 indent = 2
 short_to_long_function_def = false
 always_for_in = true
+join_lines_based_on_source = true
```

Comment:
```suggestion
```
I think just having the overlong margin should be enough to fix the call issues. Having this one here is the root of some of the problems @thofma pointed out.
@joschmitt If you apply this change, please start again with the unformatted version. Repeated formatting with different configs may change the result over time.